tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_sgd.py

```diff
@@ -191,3 +191,142 @@ def forward(self, x):
         logger.debug(f"Momentum_out pcc={out}")
 
         assert passing
+
+
+@pytest.mark.parametrize(
+    "shape",
+    ((32, 32),),  # single
```

Comment:
```suggestion
    [[32, 32]] ,  # single
```
Our coding rule? :)

Reply:
Reflecting what you mentioned, I changed the `shape` from a tuple to a list.